src/components/common/CommonHeader.vue

```diff
@@ -112,6 +112,38 @@
     <div @click.stop v-if='isSkyDriveManagerDialogShow'>
       <SkyDriveManagerDialog :show='isSkyDriveManagerDialogShow' @close='closeSkyDriveManagerDialog' />
     </div>
+    <div @click.stop v-if="isLoginDialogShow" class="login-dialog">
+      <el-dialog title="" :visible.sync="isLoginDialogShow">
+        <el-form class="login-dialog-form" :model="ruleForm" :rules="rules" ref="ruleForm">
+          <el-form-item prop="username">
+            <el-input v-model="ruleForm.username"></el-input>
+          </el-form-item>
+           <el-form-item prop="password">
+            <el-input type="password" v-model="ruleForm.password"></el-input>
+          </el-form-item>
+          <div class="login-dialog-form-operate"><a href="/wiki/find_pwd">{{$t('common.forgetPassword')}}?</a></div>
+          <el-form-item>
+            <el-button class="login-btn" type="primary" @click="login('ruleForm')">{{$t('common.login')}}</el-button>
+          </el-form-item>
+          <div class="login-dialog-form-operate_signIn">{{$t('common.noAccount')}}?<a href="/wiki/join">{{$t('common.signIn')}}</a></div>
+          <div class="login-dialog-form-three-login">
+            <div class="title">{{$t('common.usingFollowingAccount')}}</div>
+            <a>
+              <img src="https://stage.keepwork.com/wiki/assets/imgs/icon/wiki_qq.png" alt="">
```

Comment:
Put all resources needed in the project as we can as possible, please! Nobody can guarantee other websites will work forever.

Reply:
ok